Files/Interacts/Interaction.cs

```diff
@@ -427,8 +427,174 @@ public async void OpenFileLocation_Click(object sender, RoutedEventArgs e)
                 await DialogDisplayHelper.ShowDialogAsync("InvalidItemDialogTitle".GetLocalized(),
                     string.Format("InvalidItemDialogContent".GetLocalized()), Environment.NewLine, destFolder.ErrorCode.ToString());
             }
+        }/// <summary>
+         /// Navigates to a directory or opens file
+         /// </summary>
+         /// <param name="path"></param>
+         /// <param name="itemType"></param>
+         /// <param name="openSilent">Determines whether history of opened item is saved (... to Recent Items/Windows Timeline)</param>
+        public async Task<bool> OpenPath(string path, FilesystemItemType? itemType = null, bool openSilent = false)
+        // TODO: This Function is not ready yet! It is not fully merged from OpenSelectedItems() because of ListedItem restriction, halp needed plz
```

Comment:
Can you elaborate on the restriction you need help with here?

Reply:
Well, I originally tried to convert path to `ListedItem`, since no function for getting `ListedItem` from path exists, I had to rewrite entire function. There are some checks in the old function that determine whether an item is an hiddenItem, shortcutItem etc. this is not being checked in `OpenPath` function